### PR TITLE
[oneDNN] Enable oneDNN reduced precision floating point arithmetic mode with new TF_SE…

### PR DESCRIPTION
**Pull request by yimeisun123**
…T_ONEDNN_FPMATH_MODE environment setting

### CODE REVIEW

**Comment by yimeisun123** [comment]
oneDNN API info on floating point math mode - https://oneapi-src.github.io/oneDNN/group_dnnl_api_fpmath_mode.html

**Comment by gbaned** [comment]
Hi @penpornk Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @penpornk Can you please review this PR ? Thank you!

**Review by penpornk — APPROVED** [review]
Thank you for the PR and sorry for the delay!

File: tensorflow/core/util/onednn_env_vars.h
Comment (cota) [inline tensorflow/core/util/onednn_env_vars.h:1]: Please remove "using std::string"; use std::string directly. Thanks!

**Review by cota — CHANGES_REQUESTED** [review]

**Comment by bhavani-subramanian** [comment]
@cota I have addressed your review comments. Can you please take a look? Thanks.

File: tensorflow/core/util/onednn_env_vars.cc [hunk 1]
```diff
@@ -53,5 +53,15 @@ bool ThreadPoolUseCallerThread() {
   return threadpool_use_caller_thread;
 }
 
+string FPMathModeSetting() {
```
Comment (cota) [inline tensorflow/core/util/onednn_env_vars.cc:56]: Please use std::string in this file as well.

**Review by cota — COMMENTED** [review]
Please also edit tensorflow/core/util/onednn_env_vars.cc to use std::string directly.

File: tensorflow/core/util/onednn_env_vars.cc (hunk 1 above)
Comment (yimeisun123) [reply tensorflow/core/util/onednn_env_vars.cc]: Updated, thank you.

**Comment by yimeisun123** [comment]
@penpornk @cota - thank you for reviewing this PR, comments are addressed in the update. Thanks

**Review by cota — APPROVED** [review]

**Comment by yimeisun123** [comment]
1) ARM CI /build (3.10) failed and the log shows - 
`install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
error: [Errno 28] No space left on device
++ die 'build_pip_package FAILED'`

2) Py+CPP Test Suite failed, log shows following error -.
`ERROR: An error occurred during the fetch of repository 'go_sdk':`

Both are not related to this PR change.

**Pull request merged**